metis-technical-metadata-generation/src/main/java/eu/europeana/metis/technical/metadata/generation/utilities/MediaExtractorForFile.java

```diff
@@ -155,6 +194,18 @@ private void submitResource(final FileStatus fileStatus, final Set<Integer> nonR
     });
   }
 
+  private void thumbnailUpload(String resourceUrl) {
+    final TechnicalMetadataWrapper technicalMetadataWrapper = mongoDao
```

Comment:
Check for null?

Reply:
Fixed

Reply:
I actually meant to check technicalMetadataWrapper for nullity.